Error compiling v0.3.19

It's the first time I'm using macroquad,
I used ggez and gwg in the past. I get this error when trying to compile my first example:
####################################
   Compiling macroquad v0.3.19 (/home/amsoares/Programming/rust/macroquad-master)
error[E0107]: this struct takes 0 lifetime arguments but 2 lifetime arguments were supplied
   --> src/window.rs:35:41
    |
35  |     pub quad_context: &'a mut miniquad::Context<'static, 'static>,
    |                                         ^^^^^^^------------------ help: remove these generics
    |                                         |
    |                                         expected 0 lifetime arguments
    |
note: struct defined here, with 0 lifetime parameters
   --> /home/amsoares/.cargo/registry/src/github.com-1ecc6299db9ec823/miniquad-0.3.8/src/graphics.rs:632:12
    |
632 | pub struct GraphicsContext {
    |            ^^^^^^^^^^^^^^^

error[E0107]: this struct takes 0 lifetime arguments but 2 lifetime arguments were supplied
   --> src/lib.rs:153:19
    |
153 |     quad_context: QuadContext<'static, 'static>,
    |                   ^^^^^^^^^^^------------------ help: remove these generics
    |                   |
    |                   expected 0 lifetime arguments
    |
note: struct defined here, with 0 lifetime parameters
   --> /home/amsoares/.cargo/registry/src/github.com-1ecc6299db9ec823/miniquad-0.3.8/src/graphics.rs:632:12
    |
632 | pub struct GraphicsContext {
    |            ^^^^^^^^^^^^^^^

For more information about this error, try `rustc --explain E0107`.
####################################
My environment:
####################################
amsoares@ubuntu:macroquad-master$ rustup show
Default host: x86_64-unknown-linux-gnu
rustup home:  /home/amsoares/.rustup

installed targets for active toolchain
--------------------------------------

wasm32-unknown-unknown
wasm32-wasi
x86_64-unknown-linux-gnu

active toolchain
----------------

stable-x86_64-unknown-linux-gnu (default)
rustc 1.61.0 (fe5b13d68 2022-05-18)
####################################
Thanks

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -26,7 +26,7 @@ opt-level = 3
 all-features = true
 
 [dependencies]
-miniquad = { version = "0.3.7", features = ["log-impl"] }
+miniquad = { version = "=0.3.7", features = ["log-impl"] }
 quad-rand = "0.2.1"
 glam = {version = "0.14", features = ["scalar-math"] }
 image = { version = "0.23.12", default-features = false, features = ["png", "tga"] }

